src/neuroconv/tools/testing/data_interface_mixins.py

```diff
@@ -223,6 +217,26 @@ def run_custom_checks(self):
         """Override this in child classes to inject additional custom checks."""
         pass
 
+    @pytest.mark.parametrize("backend", ["hdf5", "zarr"])
+    def test_run_conversion_with_backend(self, setup_interface, tmp_path, backend):
+
+        nwbfile_path = str(tmp_path / f"conversion_with_backend{backend}_{self.test_name}.nwb")
```

Comment:
```suggestion
        nwbfile_path = str(tmp_path / f"conversion_with_backend-{backend}_{self.test_name}.nwb")
```

Reply:
you prefer `-` over `_`?

Reply:
whichever you prefer

I kind of tend to mirror what DANDI does and `_` separate thematically similar blocks, with `-` separation within a block

Reply:
I really don't have a strong preference, I will add the commit, just curios.

Reply:
Changed